roles/passenger/templates/geotaste_passenger.conf.j2

```diff
@@ -0,0 +1,106 @@
+server {
```

Comment:
what did you have to configure from the base template? it's hard to tell what you changed. Do you still need a custom passenger conf if you aren't doing the proxying?